cmd/common-methods.go

```diff
@@ -70,25 +71,59 @@ func parseKey(sseKeys string) (sse string, err *probe.Error) {
 	}
 	decodedString, e := base64.StdEncoding.DecodeString(secretValue)
 	if e != nil || len(decodedString) != 32 {
-		return "", probe.NewError(errors.New("Encryption key should be 32 bytes plain text key or 44 bytes base64 encoded key"))
+		return "", probe.NewError(errors.New("encryption key should be 32 bytes plain text key or 44 bytes base64 encoded key"))
 	}
 	return encryptString[0] + "=" + string(decodedString), nil
 }
 
 // parse and return encryption key pairs per alias.
 func getEncKeys(ctx *cli.Context) (map[string][]prefixSSEPair, *probe.Error) {
+	// SSE-S3
 	sseServer := os.Getenv("MC_ENCRYPT")
+	if sseS3 := os.Getenv("MC_SSE_S3"); sseServer != "" {
+		sseServer = sseS3
+	}
```

Comment:
You can set the encrypton as an env variable like `MC_ENCRYPT` if its not passed as a cli flag `--encrypt` but because we've decided to deprecate it, i thought it should not be removed just yet.
`MC_SSE_S3` is the way to set the env var. 
So logically, it checks if the env is set, and then override the value if you decide to pass it as a flag again

Reply:
That part I know the problem is you are looking for wrong value.

See comment https://github.com/minio/mc/pull/4606#discussion_r1236649167

Reply:
Oh, right! I would tag you in my next commit.